backend/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.9
+RUN mkdir -p /app
+WORKDIR /app
+COPY . .
+
+RUN pip install --upgrade pip
+RUN pip install -e .[dev]
+RUN pip install fastapi[all]
+RUN pip install -i https://test.pypi.org/simple/ flotilla-openapi
+
+EXPOSE 8000
+RUN pip install gunicorn
+CMD ["gunicorn", "--bind", "127.0.0.1:8000", "src/flotilla/main.py"]
```

```suggestion
CMD ["uvicorn", "--host", "0.0.0.0", "src.flotilla.main:app"]
```
We'd prefer to use `uvicorn` for the time being. Also, you have to bind to `0.0.0.0` within the container for the application to be available through the port mapping in `docker-compose.yml`